src/Renderer/ThreeExtented/GlobeControls.js

```diff
@@ -803,6 +804,10 @@ function GlobeControls(camera, domElement, engine) {
                 if (point)
                     updateSpherePicking.bind(this)(point,ptScreenClick);
 
+                if (keyD) {
+                    scene.setDetailedFeatureParameters(point, isFirstPoint);
```

Comment:
use event like this [example](https://github.com/iTowns/itowns2/blob/54ad46dd6b5e7f0d2a4363312c6b9903d7b09c9d/src/Renderer/ThreeExtented/GlobeControls.js#L787)